test/unit/runner/runner.js

```diff
@@ -88,7 +93,8 @@ describe('Runner Unit', function() {
         it('Can handle a passing and a failing test!', function() {
             let testA = new TestStub('test A', 'a', 'Hello World');
             let testB = new TestStub('test B', 'b', 'Hello Wool');
-            let tests = [testA, testB];
+            let testParams = new TestStub('test params', 'c', '2', ['1', '1']);
```

Comment:
This doesn't really test anything new. Should add some assertion to check it actually passed the parameters along.
Something like assert.strictEqual(invokerStub.invoke.thirdCall.args[3]['b'], 2) below the other asserts.
Not 100% on the syntax for sinon stub spies.
Also, should pass an assoc array as the params, as {a : 1, b : 2} instead of [1,1]

Reply:
Ok it's changed